iron-selection.d.ts

```diff
@@ -0,0 +1,12 @@
+/**
+ * DO NOT EDIT
+ *
+ * This file was automatically generated by
+ *   https://github.com/Polymer/gen-typescript-declarations
+ *
+ * To modify these typings, edit the source file(s):
+ *   iron-selection.html
+ */
+
+/// <reference path="../polymer/types/polymer.d.ts" />
+
```

Comment:
this has a few skipped functions / prototypes

Reply:
Added manual typing for this whole class since Analyzer doesn't understand it